internal/cmd/login.go

```diff
@@ -225,13 +226,44 @@ func updateInfraConfig(client *api.Client, loginReq *api.LoginRequest, loginRes
 }
 
 func oidcflow(host string, clientId string) (string, error) {
+	// find out what the authorization endpoint is
+	provider, err := oidc.NewProvider(context.Background(), fmt.Sprintf("https://%s", host))
+	if err != nil {
+		return "", fmt.Errorf("get provider oidc info: %w", err)
+	}
+
+	// claims are the attributes of the user we want to know from the identity provider
+	var claims struct {
+		ScopesSupported []string `json:"scopes_supported"`
+	}
+
+	if err := provider.Claims(&claims); err != nil {
+		return "", fmt.Errorf("parsing claims: %w", err)
+	}
+
+	scopes := []string{"openid", "email"} // openid and email are required scopes for login to work
+
+	// we want to be able to use these scopes to access groups, but they are not needed
+	wantScope := map[string]bool{
```

Comment:
making these scopes optional will allow us to login using IDPs that do not support the groups claim